src/Data/CurrentUser/verifySameWhoAmIUser.ts

```diff
@@ -35,6 +37,49 @@ export async function verifySameWhoAmIUser() {
   if (result.email === currentUserEmail) return
 
   simpleErrorToast(
-    `This link is for ${result.email ?? ''} but you are logged in as ${currentUserEmail}. Please log out first.`,
+    localizeEmailMismatch(lang, result.email ?? '', currentUserEmail),
   )
 }
+
+export function localizeExpiredMessage(language: string): string {
+  switch (language) {
+    case 'de':
+      return 'Der von Ihnen gefolgte Link ist ungültig oder abgelaufen. Bitte fordern Sie einen neuen an.'
+    case 'fr':
+      return 'Le lien que vous avez suivi est invalide ou a expiré. Veuillez en demander un nouveau.'
+    case 'pl':
+      return 'Link, któremu śledziłeś jest nieważny lub wygasł. Prosimy o poproszenie o nowy.'
+    default:
+      return 'The link you followed is invalid or has expired. Please request a new one.'
+  }
+}
+
+export function localizeFailedVerify(language: string): string {
+  switch (language) {
+    case 'de':
+      return 'Fehler beim Überprüfen des Benutzerprofils.'
+    case 'fr':
+      return 'Échec de la vérification du profil utilisateur.'
+    case 'pl':
+      return 'Nie udało się zweryfikować profilu użytkownika.'
+    default:
+      return 'Failed to verify user profile.'
+  }
+}
+
+function localizeEmailMismatch(
+  language: string,
+  linkFor: string,
+  currentUserEmail: string,
+): string {
+  switch (language) {
+    case 'de':
+      return `Dieser Link ist für ${linkFor}, aber Sie haben sich als ${currentUserEmail} eingeloggt. Bitte melden Sie sich zuerst ab.`
```

Comment:
* [ ] "sie sind als ... eingeloggt" as in the English version would cover more actual cases (e.g., a shared computer).